libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -312,6 +312,10 @@ public void onPrepareOptionsMenu(Menu menu) {
         super.onPrepareOptionsMenu(menu);
     }
 
+    public boolean hasAnyChanges() {
+        return (content.history.getHistoryEnabled() && !content.history.getHistoryList().isEmpty());
```

Comment:
I'm fairly new to Aztec so I have to ask: when would history *not* be enabled?

Reply:
Good question! I think never in WPandroid, but given it is a possibility (that Aztec allows to enable/disable the feature) I thought it would be better to future-proof it and make the check anyway

Reply:
In that case, maybe this method should be renamed to `hasHistory()`?

Reply:
sounds good 👍 , changed in 8035d97